content/cookie-policy.md

```diff
@@ -4,73 +4,50 @@ description: Learn about our cookie policy, detailing how we use cookies to enha
 ---
 
 # Cookie Policy
+-----------------
 
-This Cookies Policy explains what Cookies are and how We use them. You should read this policy so You can understand what type of cookies We use, or the information We collect using Cookies and how that information is used.
+Last updated: 4th of May 2023
 
-Cookies do not typically contain any information that personally identifies a user, but personal information that we store about You may be linked to the information stored in and obtained from Cookies. For further information on how We use, store and keep your personal data secure, see our Privacy Policy.
+This Cookie Policy explains how Kestra uses cookies on our website [kestra.io](http://kestra.io) (the "Website"). By using our Website, you consent to the use of cookies in accordance with this Cookie Policy.
```

Comment:
```suggestion
This page explains how Kestra uses cookies on [kestra.io](http://kestra.io) (i.e., the "Website"). By using the website, you consent to the use of cookies in accordance with this cookie policy.
```

Reply:
I would generally make all those lowercase: website, cookie policy -- but up to you. The main thing would be to keep it consistent. Atm, the website is sometimes named in lowercase and sometimes capitalized (Website).

Reply:
yeah I stuttered on the MD!